sentry-samples/sentry-samples-console/src/main/java/io/sentry/samples/console/Main.java

```diff
@@ -158,7 +158,7 @@ public static void main(String[] args) throws InterruptedException {
 
     // All events that have not been sent yet are being flushed on JVM exit. Events can be also
     // flushed manually:
-    // Sentry.close();
+    Sentry.close();
```

Comment:
Drawback of `async-http-client` - http client has to be explicitly closed, otherwise it will prevent JVM from shutting down.